src/maintenance/azext_maintenance/generated/commands.py

```diff
@@ -0,0 +1,74 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-statements
+# pylint: disable=too-many-locals
+
+from azure.cli.core.commands import CliCommandType
+
+
+def load_command_table(self, _):
+
+    from azext_maintenance.generated._client_factory import cf_public_maintenance_configuration
+    maintenance_public_maintenance_configuration = CliCommandType(
+        operations_tmpl='azext_maintenance.vendored_sdks.maintenance.operations._public_maintenance_configuration_opera'
+        'tions#PublicMaintenanceConfigurationOperations.{}',
+        client_factory=cf_public_maintenance_configuration)
+    with self.command_group('maintenance public-maintenance-configuration',
+                            maintenance_public_maintenance_configuration,
+                            client_factory=cf_public_maintenance_configuration, is_experimental=True) as g:
+        g.custom_command('list', 'maintenance_public_maintenance_configuration_list')
+        g.custom_show_command('show', 'maintenance_public_maintenance_configuration_show')
+
+    from azext_maintenance.generated._client_factory import cf_apply_update
+    maintenance_apply_update = CliCommandType(
+        operations_tmpl='azext_maintenance.vendored_sdks.maintenance.operations._apply_update_operations#ApplyUpdateOpe'
+        'rations.{}',
+        client_factory=cf_apply_update)
+    with self.command_group('maintenance apply-update', maintenance_apply_update, client_factory=cf_apply_update,
```

Comment:
the command is changed, before it is "applyupdate", but now it 's "apply-update", is this breaking change expected?

Reply:
regenerated with existing command names